Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Tcc_Feature.dlt

```diff
@@ -0,0 +1,29 @@
+## @file
+#
+#  TCC feature Configuration Delta File.
+#
+#  Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+#
+
+# Enable TCC config data
+TCC_CFG_DATA.TccEnable   | 1
+TCC_CFG_DATA.TccTuning   | 1
+TCC_CFG_DATA.TccSoftSram | 1
+TCC_CFG_DATA.TccErrorLog | 1
+
+# Enable Timed GPIO
+SILICON_CFG_DATA.EnableTimedGpio0 | 1
+SILICON_CFG_DATA.EnableTimedGpio1 | 1
+
+# Enable RTCM from the boot option
+BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 24
+BOOT_OPTION_CFG_DATA_1.ImageType_1 | 0x1E
+
+BOOT_OPTION_CFG_DATA_2.BootFlags_2 | 24

```

Comment:
BOOT_OPTION_CFG_DATA_2 is for RTCM, the boot flags doesn't matter

Reply:
done